src/nft/components/profile/view/WalletAssetDisplay.tsx

```diff
@@ -65,7 +65,7 @@ export const WalletAssetDisplay = ({ asset, isSellMode }: { asset: WalletAsset;
           width="full"
```

Comment:
This file is basically replaced with `ViewMyNftsAsset` yeah?

Reply:
yes! I renamed it to be the same convention as "CollectionAsset", which handles the cards in Collection page.